Sources/Plasma/PubUtilLib/plInputCore/plInputDevice.h

```diff
@@ -175,6 +175,7 @@ class plMouseDevice : public plInputDevice
     uint32_t  GetButtonState() { return fButtonState; }
     float GetCursorOpacity() { return fOpacity; }
     void SetDisplayResolution(float Width, float Height);
+    void SetScale(float Scale);
```

Comment:
Wondering if maybe this should be SetDisplayScale instead.

Reply:
I don't have a strong preference either way, but `SetDisplayScale` seems consistent with `SetDisplayResolution`